cal/cal/static/js/services.js

```diff
@@ -238,10 +238,11 @@ analyticsApp.service('QueryService', function() {
     var ctrlDetails = [];
     var maxYValue = 0;
     var events = [];
+    var movingAverage = [];
     var xLabels = [];
     var yLabels = [];
-    for (var i = 0; i < data.length; i++) {
-      var event = data[i];
+    for (var i = 0; i < data[0].length; i++) {
```

Comment:
you can do for (var i = 0, var j = 0; i < data.length, j < data[0].length; i++, j++)